components/asio/examples/asio_chat/main/idf_component.yml

```diff
@@ -1,4 +1,6 @@
 dependencies:
   ## Required IDF version
   idf: ">=5.0"
-  espressif/asio: ">=1.0.1"
+  espressif/asio:
+    version: "^1.14.1"
```

Comment:
I have changed >= to ^ just in case there is a new major version with an incompatible API in the future.

>= would still pick up that new version, and projects created from such examples will start failing.

Reply:
makes sense, thanks for changing this!